src/TodoList/TodoList.js

```diff
@@ -2,37 +2,33 @@ import React, { Component } from 'react';
 import './TodoAll.css';
 import { TodoForm } from './TodoForm/TodoForm.js'
 import { TodoItem } from './TodoItem/TodoItem.js'
-import { FaArrowRight, FaArrowLeft } from 'react-icons/lib/fa';
 
 export class TodoList extends Component {
-    constructor(props) {
-    super(props);
-  }  
-    
-  getArrowButton = (todoStatus) => {
-    if (todoStatus === 'To do'){
-      return <FaArrowRight />;
-    } else if (todoStatus === 'Done'){
-      return <FaArrowLeft />;
-    }
-  }
-    
+
+  progressArrow = (index, lastIndex) => {
+    return (index !== lastIndex) ? true : false; 
+  } 
+
+  regressArrow = (index) => {
+    return (index !== 0) ? true : false;
+  } 
+
   render() {
     return (
       <div className="todo-list">
         <TodoForm 
           placeholder={this.props.list}
           todoStatus={this.props.list}
-          addTodoHandler={this.props.addTodoHandler}
-        />
+          addTodoHandler={this.props.addTodoHandler}/>
         {this.props.todos.map(todo => (
           <TodoItem 
             key={todo.id}
             todo={todo}
-            todoStatus={this.props.list}
-            getArrowButton={this.getArrowButton(this.props.list)}
-            changeTodoPlace={this.props.changeTodoPlace}
-            removeTodo={this.props.removeTodo}
+            progressArrow={this.progressArrow(this.props.index, this.props.lastIndex)}
```

Comment:
This is not a good idea. You can either create two variables before return statement `const withProgressArrow = this.props.index !== this.props.lastIndex` (and same for regress).
But in my opinion a better solution is to pass two props to `TodoList` component:
`isLastList` and `isFirstList`, and use this props: `withProgressArrow={!isLastList}` etc.